pkg/controller/controller.go

```diff
@@ -0,0 +1,18 @@
+package controller
+
+import (
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+)
+
+// AddToManagerFuncs is a list of functions to add all Controllers to the Manager
+var AddToManagerFuncs []func(manager.Manager) error
```

Comment:
Not a part of this PR, but once we have it, you can add a controller managing a resource by calling `operator-sdk add controller --api-version=app.example.com/v1alpha1 --kind=AppService`